Makefile

```
@@ -14,7 +14,13 @@ deps: sync
 run: deps
 	uv run --env-file .env -- langgraph dev --allow-blocking --debug-port 2025
 
-smee: smee-mPfw041ExPaQji0
+smee:
```

checkmake (0.2.2)

[warning] Target body for "smee" exceeds allowed length of 5 (6).

(maxbodylength)